app/(storyblok)/[[...slug]]/page.tsx

```diff
@@ -86,7 +86,14 @@ async function getStoryData(params: { slug: string[] }) {
   };
 
   try {
-    const story = await storyblokApi.get(`cdn/stories/${slug}`, sbParams);
+    const story = await unstable_cache(
```

Comment:
![Screenshot 2023-12-20 at 13 35 36](https://github.com/SU-SWS/ood-giving-campaign/assets/550602/3c48bb19-fec4-48e1-859f-dce380ac1354)
![Screenshot 2023-12-20 at 13 48 04](https://github.com/SU-SWS/ood-giving-campaign/assets/550602/dbea2ac8-82df-430d-a80c-c928b99b48bd)

4mb of page data????

Reply:
What?? 👀

Reply:
Did a lighthouse run the other day and performance was like 86% 👀

Reply:
Yikes, yeah.

All of the relations (Nested blocks, stories, media, etc) are all fully resolved and well, they add up.